docs/home.md

```diff
@@ -1,56 +1,31 @@
 # Overview of TechPass
 
-TechPass is the Identity & Access Management (IAM) and Single Sign-on (SSO) solution for the following [Singapore Government Technology Stack (SGTS) services](#sgts-services-integrated-with-techpass).
+TechPass is the Identity & Access Management (IAM) and Single Sign-on (SSO) solution for the following Singapore Government Technology Stack (SGTS) services.
 
-<a id="sgts-services-integrated-with-techpass">
 
-**SGTS services integrated with TechPass**
-
-</a>
+## SGTS services integrated with TechPass
 
 Following are the list of SGTS services that uses TechPass as their IAM service.
 
 - Container Stack(CStack): CStack is a cloud-based container development and hosting platform.
```

Comment:
CStack's value doesn't stand out based on this description, unlike the others. Just my 2 cents.

Reply:
Updated to: This is a base Layer product of the SGTS tech stack offering a centralised, secure, and fully managed development platform for containers and Kubernetes.